### PR TITLE
templates: Fix redirection to clones on advanced restart actions

### PR DESCRIPTION
**Pull request by okurz**
Related progress issue: https://progress.opensuse.org/issues/69979

### CODE REVIEW

**Comment by kraih** [comment]
Some ui tests might be good, considering how JavaScript heavy the code path is.

File: t/ui/26-jobs_restart.t [hunk 1]
```diff
@@ -142,7 +142,7 @@ subtest 'restart job from info panel in test results' => sub {
     subtest 'successful restart' => sub {
         is($driver->get('/tests/99946'), 1, 'go to job 99946');
         update_last_job_id;
-        $driver->find_element('button > #restart-result')->click();
+        $driver->find_element('#restart-result')->click();
```
Comment (kalikiana) [inline t/ui/26-jobs_restart.t:145]: You're changing it here. But the test wasn't failing before either. Please convince me why this should work now w/o other changes, or extend the test accordingly

**Review by kalikiana — CHANGES_REQUESTED** [review]

File: t/ui/26-jobs_restart.t (hunk 1 above)
Comment (okurz) [reply t/ui/26-jobs_restart.t:149]: Added according tests for the redirection after clicking the advanced menu entries same like on the default action where it's already tested

**Comment by okurz** [comment]
> Some ui tests might be good, considering how JavaScript heavy the code path is.

Extended tests to cover that

**Review by kalikiana — APPROVED** [review]

**Review by kraih — APPROVED** [review]

**Pull request merged**